apps/addons/templates/addons/impala/addon_hovercard.html

```diff
@@ -4,9 +4,9 @@
     <div class="icon">
       <a href="{{ details_link }}">
       {% if lazyload %}
-        <img src="{{ addon.icon_url }}" alt="">
+      <img src="{{ addon.icon_url }}" alt="{{ _('Icon for') }} {{ addon.name }}">
```

Comment:
When localizing, we sadly can't do it that way (which would be much simpler), because the order of the words in a sentence can vastly vary between different locales.

The correct way to do that is https://github.com/Nolski/olympia/blob/alt-a11y-main-page/apps/addons/templates/addons/impala/details.html#L99
```{{ _('Icon for {0}')|fe(addon.name) }}```